Training/Program.cs

```diff
@@ -1,7 +1,33 @@
-namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+string inputStr = InputValidityChecker ("Enter a string: ", "Enter a valid string. Empty string is not allowed");
+string inSplChar = InputValidityChecker("Mention the special character to be swapped: ", "Enter a valid special character. Entering empty special character is not allowed");
+Console.Write ("Mention the order: ");
+string order = Console.ReadLine ();
+SortAndSwapSplChars (inputStr, inSplChar, order);
+
+static void SortAndSwapSplChars (string inputStr, string inSplChar, string order) {
+   char splChar = char.Parse (inSplChar);
+   List<char> tempList = inputStr.ToList ();
+   if (inputStr.Contains (splChar)) {
+      tempList.RemoveAll (ch => ch == splChar);
+      tempList.Sort ();
+      tempList.ToArray ();
+   } else tempList.Sort ();

```

Comment:
tempList.Sort () is done in both if and else. Can be included at the end of if statement.